pkg/sql/sqlbase/table_col_set.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2020 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package sqlbase
+
+import "github.com/cockroachdb/cockroach/pkg/util"
+
+// TableColSet efficiently stores an unordered set of column ids.
+type TableColSet struct {
```

Comment:
What is the reason for wrapping util.FastIntSet here? As far as I can tell all the methods just call into `util.FastIntSet`.

Reply:
It's so it's strongly typed, which makes it evident what it stores and you don't have to cast ColumnIDs to ints and back.

Reply:
I guess, but it also feels like a good amount of duplication.

Reply:
After working without a wrapper for opt.ColSet for a few releases (until Go implemented mid-stack inlining) I can tell you that the duplication is worth it.

Reply:
Sounds good -- i haven't had to use this extensively to really have an opinion.

Reply:
I think the bit of duplication is worth it for the clarity and conciseness it provides when using `TableColSet`.